notifications/templates/email/local_units/local_unit.html

```diff
@@ -0,0 +1,43 @@
+<p>Dear {{full_name}} {% include "includes/logo.html" %}</p>
```

Comment:
This is weird. What does `logo.html` introduce? Can this be splitted on multiple lines?

Reply:
Yes, this file needs to be refactored with correct styling and structure. Someone from frontend will need to help here.

@susilnem will add a preview view in backend for local develop run which can be used for developer to make required changes.

We will also need to add a proper base template for emails as well